internal/cephfs/volume.go

```diff
@@ -88,8 +88,9 @@ func (vo *volumeOptions) getSubVolumeInfo(ctx context.Context, volID volumeID) (
 		if errors.Is(err, rados.ErrNotFound) {
 			return nil, ErrVolumeNotFound
 		}
-		// In case the error is other than invalid command return error to the caller.
-		if !strings.Contains(err.Error(), invalidCommand) {
+		// In case the error is invalid command return error to the caller.
+		var invalid *fsAdmin.NotImplementedError
+		if errors.As(err, &invalid) {
```

Comment:
same here, `&invalid` is now a poiner-to-a-pointer-to-a-fsAdmin.NotImplementedError

Reply:
reference is taken from https://golang.org/pkg/errors/#example_As hope this is fine.

Reply:
I am not sure, existing usage in the current code (including `vendor/`) does not use a double pointer... This should be verified with a small test program.

Reply:
remove double-pointer and updated the error check